<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionForm</class>
 <widget class="QWidget" name="PositionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Posición</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="currentPositionLabel">
        <property name="text">
         <string>Posición actual: ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="valueLabel">
        <property name="text">
         <string>Posición deseada: ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="holdPushButton">
       <property name="text">
        <string>Capturar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="acceptPushButton">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
